setup.cfg

```
@@ -0,0 +1,74 @@
+[metadata]
+name = shiver
+version = 0.0.1
+distance = 0
+description = Spectroscopy Histogram Visualizer for Event Reduction
+long_description = file: README.md, LICENSE
+long_description_content_type = text/markdown
+url = https://github.com/neutrons/Shiver/
+license = GPL3.0
+
+[options]
+package_dir =
+    =src
+include_package_data = True
+packages = find:
+python_requires >= 3.8
+install_requires =
+    mantidworkbench
+    pyoncat == 1.4.1
```

Change to "fuzzy version" so you can get newer patch releases if they come out
```suggestion
    pyoncat == 1.4
```

---

"No matching distribution found" error without the subversion

---

I don't know why we were using 1.4.1, the newest pyoncat version is 1.6.1. https://oncat.ornl.gov/packages/ We should move to that.

---

done